cmd/ontap-cvo.go

```diff
@@ -128,7 +176,8 @@ func getOnPrem(ownerEmail string, client *kubernetes.Clientset) (string, bool) {
 	// Step 0 Get the App Registration Info
 	// Don't forget to create a secret in the namespace for authentication with azure in the namespace
 	// for me in aaw-dev its under jose-matsuda
-	secret, err := client.CoreV1().Secrets("netapp").Get(context.Background(), "netapp-regi-secret", metav1.GetOptions{})
+	// TODO change to das? for the location of secrets
```

Comment:
Is there a reason for wanting to move from _netapp_ namespace to _das_ namespace? Is this just because that is where the controller lives?

Reply:
more for because we have controllers in das to begin with